src/plugins/workspace/public/components/workspace_detail/workspace_detail.test.tsx

```diff
@@ -16,7 +16,7 @@ import { WorkspaceFormProvider, WorkspaceOperationType } from '../workspace_form
 import { DataSourceConnectionType } from '../../../common/types';
 import * as utilsExports from '../../utils';
 import { IntlProvider } from 'react-intl';
-
+import { of } from 'rxjs';
```

Comment:
Nit: let's keep the empty line
```suggestion
import { of } from 'rxjs';

```